internal/message/view.go

```diff
@@ -637,6 +637,18 @@ func (v *View) ScrollToMessage(id discord.MessageID) bool {
 	return true
 }
 
+// AddReaction adds an reaction to the message with the given ID.
+func (v *View) AddReaction(id discord.MessageID, emoji discord.APIEmoji) {
+	state := gtkcord.FromContext(v.ctx)
+
+	emoji = discord.APIEmoji(gtkcord.SanitizeEmoji(string(emoji)))
+
+	error := state.React(v.chID, id, emoji)
```

Comment:
This will block the main thread. Do it in a goroutine:

```go
go func() {
  if err := state.React(...); err != nil {
    app.Error(state.Context(), errors.Wrap(err, "cannot react")) 
  }
}()
```